src/components/TradeWidget/OrderValidity.tsx

```diff
@@ -58,137 +82,211 @@ const Wrapper = styled.div`
   }
 `
 
-// const InputBox = styled.div`
-//   display: flex;
-//   flex-direction: column;
-//   align-items: stretch;
-//   flex-grow: 1;
-
-//   margin-left: 1em;
-
-//   > div {
-//     display: flex;
-//     justify-content: center;
-//     align-items: center;
-//     > input[type='radio'] {
-//       width: 20%;
-//     }
-//   }
-
-//   .radio-container {
-//     display: grid;
-//     grid-template-columns: 1fr 1fr;
-//   }
-
-//   input {
-//     margin: 0;
-//     width: 100%;
-
-//     &.error {
-//       // box-shadow: 0 0 0.1875rem #cc0000;
-//       border-color: #ff0000a3;
-//     }
-
-//     &.warning {
-//       // box-shadow: 0 0 0.1875rem #ff7500;
-//       border-color: orange;
-//     }
-
-//     &:disabled {
-//       box-shadow: none;
-//     }
-//   }
-// `
-
-// const WalletDetail = styled.div`
-//   display: flex;
-//   justify-content: space-between;
-//   font-size: 0.75em;
-
-//   .success {
-//     color: green;
-//     text-decoration: none;
-//   }
-
-//   &.error,
-//   &.warning {
-//     margin: 0 0 1em 0;
-//   }
-
-//   &.error {
-//     color: red;
-//   }
-//   &.warning {
-//     color: orange;
-//   }
-// `
-
-// function validatePositive(value: string): true | string {
-//   return Number(value) == 0 || Number(value) >= 5 || 'Invalid expiration time'
-// }
+const OrderValidityInputsWrapper = styled.div<{ $visible: boolean }>`
+  opacity: ${({ $visible }): string => ($visible ? '1' : '0')};
```

Comment:
If you want to hide completely while preserving layout use `visibility: hidden`

Reply:
Also `opacity: 0` doesn't prevent clicks, so you can't click everything behind this fully opaque element (can't click Token selector)

Reply:
Why do you render that modal when not in use for that matter?

Reply:
no i dont, i like the fade effect

Reply:
You can't just use opacity
you can't click on what's behind right now
Try clicking USDC token selector